From 50663dc149619aef23bcb43e6fdf48bc60487374 Mon Sep 17 00:00:00 2001
From: "Darrick J. Wong" <darrick.wong@oracle.com>
Date: Thu, 25 Jan 2018 13:55:01 -0600
Subject: [PATCH] xfsprogs: update dead urls

Since oss.sgi.com is dead and xfs.org is slowly migrating to
xfs.wiki.kernel.org, update all the documentation links to point to the
current landing pads.

Signed-off-by: Darrick J. Wong <darrick.wong@oracle.com>
Reviewed-by: Eric Sandeen <sandeen@redhat.com>
Signed-off-by: Eric Sandeen <sandeen@sandeen.net>
---
 README           | 2 +-
 debian/control   | 6 +++---
 debian/copyright | 2 +-
 debian/watch     | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

Index: xfsprogs-4.5.0/README
===================================================================
--- xfsprogs-4.5.0.orig/README
+++ xfsprogs-4.5.0/README
@@ -9,4 +9,4 @@ and references to other XFS manual pages
 
 For more information and details on how to contribute to the
 XFS project see the web pages at:
-	http://oss.sgi.com/projects/xfs/
+	https://xfs.wiki.kernel.org/
Index: xfsprogs-4.5.0/debian/control
===================================================================
--- xfsprogs-4.5.0.orig/debian/control
+++ xfsprogs-4.5.0/debian/control
@@ -24,7 +24,7 @@ Description: Utilities for managing the 
  Btrees (directories, extents, free space) to aid both performance
  and scalability.
  .
- Refer to the documentation at http://oss.sgi.com/projects/xfs/
+ Refer to the documentation at https://xfs.wiki.kernel.org/
  for complete details.
 
 Package: xfslibs-dev
Index: xfsprogs-4.5.0/debian/copyright
===================================================================
--- xfsprogs-4.5.0.orig/debian/copyright
+++ xfsprogs-4.5.0/debian/copyright
@@ -1,7 +1,7 @@
 This package was debianized by Nathan Scott nathans@debian.org on
 Sun, 19 Nov 2000 07:37:09 -0500.
 
-It can be downloaded from ftp://oss.sgi.com/projects/xfs/download/
+It can be downloaded from https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/
 
 Copyright:
 
Index: xfsprogs-4.5.0/debian/watch
===================================================================
--- xfsprogs-4.5.0.orig/debian/watch
+++ xfsprogs-4.5.0/debian/watch
@@ -1,3 +1,3 @@
 version=3
 opts=uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/ \
-ftp://oss.sgi.com/projects/xfs/cmd_tars/xfsprogs-(.+)\.tar\.gz
+https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-(.+)\.tar\.xz
Index: xfsprogs-4.5.0/libxfs/util.c
===================================================================
--- xfsprogs-4.5.0.orig/libxfs/util.c
+++ xfsprogs-4.5.0/libxfs/util.c
@@ -690,7 +690,7 @@ libxfs_fs_repair_cmn_err(int level, xfs_
 	fprintf(stderr, "  This is a bug.\n");
 	fprintf(stderr, "%s version %s\n", progname, VERSION);
 	fprintf(stderr, "Please capture the filesystem metadata with "
-			"xfs_metadump and\nreport it to xfs@oss.sgi.com.\n");
+			"xfs_metadump and\nreport it to linux-xfs@vger.kernel.org.\n");
 	va_end(ap);
 }
 
Index: xfsprogs-4.5.0/man/man8/xfs_mdrestore.8
===================================================================
--- xfsprogs-4.5.0.orig/man/man8/xfs_mdrestore.8
+++ xfsprogs-4.5.0/man/man8/xfs_mdrestore.8
@@ -51,4 +51,4 @@ returns an exit code of 0 if all the met
 .BR xfs (5)
 .SH BUGS
 Email bug reports to
-.BR xfs@oss.sgi.com .
+.BR linux-xfs@vger.kernel.org .
Index: xfsprogs-4.5.0/man/man8/xfs_metadump.8
===================================================================
--- xfsprogs-4.5.0.orig/man/man8/xfs_metadump.8
+++ xfsprogs-4.5.0/man/man8/xfs_metadump.8
@@ -154,4 +154,4 @@ command.
 .BR xfs (5)
 .SH BUGS
 Email bug reports to
-.BR xfs@oss.sgi.com .
+.BR linux-xfs@vger.kernel.org .
